demes/demes.py

```diff
@@ -143,21 +149,113 @@ def __attrs_post_init__(self):
             raise ValueError("source and dest cannot be the same deme")
 
 
+@attr.s(auto_attribs=True)
+class Split:
+    """
+    Parameters for a split event, in which a deme ends at a given time and
+    contributes ancestry to an arbitrary number of descendant demes. Note
+    that there could be just a single descendant deme, in which case ``split``
```

Comment:
We could enforce > 1 child deme?

Reply:
I think that makes sense for a "split". Should we enforce that if two demes are defined (parent/child), and no other demes are involved (i.e., just a "hand-off" from one deme to the next), that users need to define that as a single deme?